cmd/ceph-cosi-driver/cmd.go

```diff
@@ -69,6 +68,12 @@ func init() {
 		driverAddress,
 		"path to unix domain socket where driver should listen")
 
+	stringFlag(&driverPrefix,
+		"driverPrefix",
```

Comment:
Should this be `driver-previx` to match the `driver-addr` flag above? I usually don't see CLI flags with camel case.

```suggestion
		"driver-prefix",
```